controllers/company.go

```diff
@@ -15,7 +15,8 @@ import (
 
 // CompanyLoginInfo is the login info for a company
 type CompanyLoginInfo struct {
-	Company string `json:"company"`
+	Email string `json:"email"`
+	Password string `json:"password"`
```

Comment:
The algorithm for bcrypt automatically salts the password. Do you want us to have another salt? :) 
https://github.com/golang/crypto/blob/master/bcrypt/bcrypt.go